strings/merge_string_checker.py

```diff
@@ -0,0 +1,38 @@
+"""
+At a job interview, you are challenged to write an algorithm to check if a given stringing, s, can be formed from two other stringings, part1 and part2.
+The restringiction is that the characters in part1 and part2 are in the same order as in s.
+The interviewer gives you the following example and tells you to figure out the rest from the given test cases.
+'codewars' is a merge from 'cdw' and 'oears':
+s:  c o d e w a r s   = codewars
+part1:  c   d   w         = cdw
+part2:    o   e   a r s   = oears
+"""
+
+# Recursive Solution
+def is_merge(s, part1, part2):
```

Comment:
Give the functions diffrent names. For example ```is_merge_rec``` for the recursive version. And ```is_merge_iter``` for the iterative version. This makes it simpler for writing tests.